src/EventStore.Common/Log/logconfig.json

```diff
@@ -1,10 +1,22 @@
 {
-  "Logging": {
-	"LogLevel": {
-	  "Default": "Debug",
-	  "System": "Warning",
-	  "Microsoft": "Warning",
-	  "Grpc": "Fatal"
+	"Logging": {
+		"LogLevel": {
+			"Default": "Debug",
+			"System": "Warning",
+			"Microsoft": "Warning",
+			"Grpc": "Fatal"
+		}
+	},
+	"Serilog": {
```

Comment:
If we include a `Serilog` section this means that the default in code logging configuration will get overridden. This is something we definitely do not want.

Reply:
I am not sure why I changed this file. I didn't have any change planned for it, good catch, I just need to revert it.